client/my-sites/scan/wpcom-upsell.tsx

```diff
@@ -77,17 +77,17 @@ export default function WPCOMScanUpsellPage(): ReactElement {
 				</p>
 				<PromoCardCTA
 					cta={ {
-						text: translate( 'Upgrade to Business Plan' ),
+						text: translate( 'Upgrade to Pro Plan' ),
```

Comment:
This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052